ui/src/translations/fr.json

```diff
@@ -124,5 +123,10 @@
   "add_solution_to_environment": "Ajout d'une solution à l'environnement {environment}",
   "version_env": "Versions",
   "add_solution": "Ajout de la solution",
-  "advanced_monitoring": "Supervision Avancée"
+  "advanced_monitoring": "Supervision Avancée",
```

Comment:
`Supervision avancée` (TBC, but I don't thing the French rules require to capitalize every word in a title, unlike the English ones)